azure/azure-release-template.yml

```diff
@@ -56,12 +56,13 @@ extends:
           fi
 
           export APIGEE_ENVIRONMENT=$(APIGEE_ENVIRONMENT)
-
           export PACT_BROKER_URL=$(PACT_BROKER_URL)
           export PACT_BROKER_BASIC_AUTH_USERNAME=$(PACT_BROKER_BASIC_AUTH_USERNAME)
           export PACT_BROKER_BASIC_AUTH_PASSWORD=$(PACT_BROKER_BASIC_AUTH_PASSWORD)
-          export PACT_PROVIDER_URL=https://$APIGEE_ENVIRONMENT.api.service.nhs.uk/$SERVICE_BASE_PATH
+          export PACT_BROKER_NEXT_URL=$(PACT_BROKER_NEXT_URL)
```

Comment:
Where are these used?

Reply:
By which I mean `PACT_BROKER_NEXT_URL` and `PACT_BROKER_NEXT_TOKEN`

Reply:
Ah, I think I've now seen the answer for `PACT_PROVIDER_URL`, but the two `PACT_BROKER_NEXT` ones appear to be unused

Reply:
I added these to be able to test out using a different broker to diagnose stability issues without breaking other prs depending on the current broker, but can clean up if you'd rather we didn't have placeholder variables, these are also referenced in publish and verify

Reply:
If it's going to be useful in future, leave it in. I just thought it might be vestigial code